Selenium/selenium8.rb

```diff
@@ -0,0 +1,30 @@
+require 'selenium-webdriver'
+
+# Open Chrome Browser
+
+Selenium::WebDriver::Chrome::Service.driver_path = 'C:\Users\neera.yadav\Documents\BrowserDrivers\chromedriver-win64\chromedriver-win64\chromedriver.exe'
+driver = Selenium::WebDriver.for :chrome
+
+# Navigate to a website
+driver.get 'https://www.globalsqa.com/demo-site/sliders/#Steps'
+
+slider_handle = driver.find_element(class: 'ui-slider-handle')
```

Comment:
The slider is present in an iframe and you need to switch to it first before getting the slider element

Reply:
Yes tried but its not able to select the element